lib/client.js

```diff
@@ -266,3 +275,40 @@ function _processHTTPError(error) {
   }
   return {cause, httpStatusCode};
 }
+
+async function _signRequest({keyId, url, json, ledgerNodeId}) {
```

Comment:
So I think this is a bit tangled to have both the `ledgerNodeId` and the `keyId` here. I think we should pass in the signer API here instead of the `ledgerNodeId` and fetch it externally.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/249/commits/e90e694686bc49f31c52bbdfa63fe6a3d3dc6f37